integration-tests/package.json

```diff
@@ -1,38 +1,39 @@
 {
   "private": true,
   "name": "@commercetools/integration-tests",
+  "version": "0.0.0",
```

Comment:
Could we please remove this one again? Otherwise, Lerna will keep bumping the tests every time we run release.

https://github.com/lerna/lerna/issues/1942

Reply:
Yarn was complaining that it didn't have a version. Also, the package is private, so there is no need to worry about the version here.

Reply:
Strange yarn :( It's just a bit ugly and potentially confusing for newcomers to keep updating that package.

Reply:
It's fine, it's part of the monorepo/workspace.